clients/java/src/main/java/io/camunda/client/api/command/CompleteJobCommandStep1.java

```diff
@@ -72,45 +72,86 @@ public interface CompleteJobCommandStep1
   CompleteJobCommandStep1 variable(String key, Object value);
 
   /**
-   * The result of the completed job as determined by the worker.
+   * Initializes the job result to allow corrections or a denial to be configured.
+   *
+   * <p>This method is used to apply changes to user task attributes (such as {@code assignee},
+   * {@code priority}, {@code dueDate}, and so on) or explicitly deny a user task operation.
```

Comment:
🔧 I suggest to use lifecycle transition rather than operation

🔧 There are more mentions of operation in this PR. Please consider all of them.

```suggestion
   * <p>This method is used to apply changes to user task attributes (such as {@code assignee},
   * {@code priority}, {@code dueDate}, and so on) or explicitly deny a user task lifecycle transition.
```